proto/README.md

```diff
@@ -20,7 +20,36 @@ The canonical Protobuf definitions live in the `proto` folder of the relevant
 release branch of CometBFT. These definitions are published to the [Buf
 registry][buf] for integrators' convenience.
 
-## Why does CometBFT use `tendermint` Protobuf definitions?
+The Protobuf files are organized under two domains: `cometbft` and `tendermint`.
+The `cometbft.*` packages use version suffixes to let application developers
+target versions of the protocols as they have evolved between CometBFT releases.
+
+## Which CometBFT release does each package belong to?
+
+By the 1.0.0 release, the entire set of Protobuf definitions used by CometBFT
+is published in packages suffixed with `.v1`. Earlier revisions of the
+definitions, where they differed, are provided alongside in `.v1beta`_N_
+packages, corresponding to 0.x releases as follows:
+
+| Domain          | 0.34      | 0.37      | 0.38      |
+|-----------------|-----------|-----------|-----------|
+| `abci`          | `v1beta1` | `v1beta2` | `v1beta3` |
+| `blocksync`     |           | `v1beta1` | `v1`      |
+| `consensus`     | `v1beta1` | same      | same      |
+| `crypto`        | `v1`      | same      | same      |
+| `libs/bits`     | `v1`      | same      | same      |
+| `mempool`       | `v1`      | same      | same      |
+| `p2p`           | `v1`      | same      | same      |
+| `privval`       | `v1beta1` | same      | `v1`      |
+| `rpc/grpc`[^1]  | `v1beta1` | `v1beta2` | `v1beta3` |
+| `state`         | `v1beta1` | `v1beta2` | `v1beta3` |
+| `statesync`     | `v1`      | same      | same      |
+| `types`         | `v1beta1` | `v1beta2` | `v1`      |
+| `version`       | `v1`      | same      | same      |
+
+[^1]: Retired in 1.0
+
+## Why does CometBFT provide `tendermint` Protobuf definitions?
```

Comment:
Please remove this section - I think it's already explained earlier in the readme.